scholarx/2023/styles.css

```diff
@@ -0,0 +1,125 @@
+/*Add custom styles for the scholarx 2020 page here*/
+.timeline {
```

Comment:
can't we use the same CSS file in 2022 page?

Reply:
I was assuming we may have to move the `/2022` to `/past-programs`. Do we?

Reply:
yeah but let's link the same style sheet. Otherwise, the code is duplicated. @Madhawa97

Reply:
roger that!